src/pages/students/create-student/CreateStudent.tsx

```diff
@@ -0,0 +1,112 @@
+import type { CreateStudentRequest } from "../../../app/types/requests/CreateStudentRequest.type";
+import {TitleContainer,
+        Title,
+        MainContainer
+}from "./CreateStudent.styles";
+
+export interface StudentDataProps  {
+    name:string;
+    lastName:string;
+    dni:string;
+    cellphoneNumber: string;
+    birthday: string;
+    pathologies?: string | null;
+}
+
+export interface PaymentDataProps {
+    paymentPlanName:string;
+    extraClasses?: number|null|undefined;
+    classPrice?:number|null|undefined;
+    paymentDay?: number |undefined;
+}
+
+export interface PlanDataProps {
+    planId: string;
+}
+
+
+function CreateStudent() {
+  const [studentData, setStudentData] = useState<StudentDataProps | undefined>(undefined);
+  const [paymentData, setPaymentData] = useState<PaymentDataProps|undefined>(undefined);
+  const [planData, setPlanData] = useState <PlanDataProps|undefined>(undefined)
+
+  const [count, setCount] = useState<number>(1)
+
+  const [createStudent]=useCreateStudentMutation();
+
+  const {userId}= useAuthentication()
+
+    const handleStudentDataForm = (data: StudentDataProps) => {
+        setStudentData(data);
+        setCount(2);
+    }
+
+    const handlePaymentDataForm = (data:PaymentDataProps)=>{
+        setPaymentData(data);
+        setCount(3)
+    }
+
+    const navigate = useNavigate();
+
+    const handlePlanDataForm = (data:PlanDataProps)=>{
+        
+        setPlanData(data)
+
+        if(!userId || !studentData || !paymentData || !planData) return;
+
+        const createStudentRequest: CreateStudentRequest = {
+            ...studentData,
+            ...paymentData,
+            ...planData,
+            admissionDate: new Date(),
+            userId
+        }
+        createStudent(createStudentRequest)
+            .unwrap()
+            .then(() => {
+              navigate("/home");
+              toast.success("El estudiante ha sido registrado");
+            })
+            .catch(() => {
+              toast.error("Ha ocurrido un error en la creación del estudiante");
+            });
+          };
+
+    const stepBack = ()=>{
+        setCount(count - 1)
+    }
+
+    return (
+        <MainContainer>
+            <TitleContainer>
+                    <Title>REGISTRAR NUEVO ALUMNO</Title>
+            </TitleContainer>
+            {count == 1 && <StudentData 
+                    createStudentCall={handleStudentDataForm}
+                    studentData={studentData} 
+                /> 
+            }
+            {count == 2 && <PaymentData
+                        createStudentCall={handlePaymentDataForm}
+                        paymentData={paymentData}
+                        stepBack={stepBack}
+                />   
+            }
+            {count == 3 && <PlanData
+                        createStudentCall={handlePlanDataForm}
+                        planData={planData}
+                        stepBack={stepBack}
+                />   
+            }
```

Comment:
Esto lo podriamos mejorar si utilizaramos un mapa que, segun el valor del contador, te retorne el fomrulario que se debe renderizar


```suggestion
            { FormComponentMap.get(count) }
```

Y el mapa lo podrias definir asi

```
    const FormComponentMap: Map<number, JSX.Element> = new Map([
        [1, <StudentData onNext={handleStudentDataForm} onBack={() => navigate("/home")} data={studentData} />],
        [2, <PaymentData onNext={handlePaymentDataForm} onBack={stepBack} data={paymentData}/>],
        [3, <PlanData onNext={handlePlanDataForm} onBack={stepBack} data={planData}/>]
    ])
```

Sobre el mapa, fijate que los formularios ya estan definidos usando la nueva interfaz, y ademas algo que no tenia el `<StudentData/>` que agregue fue el onBack que, si le das a cancelar te redirija al home